packages/client/src/shared/components/PublicLayout/PublicLayout.tsx

```diff
@@ -0,0 +1,17 @@
+import './PublicLayout.css';
+import { Layout } from 'antd';
+import { Outlet } from 'react-router-dom';
+
+function PublicLayout({ children }: { children?: React.ReactNode }) {
+  const { Content } = Layout;
+
+  return (
+    <Layout className="public-layout">
+      <div className="public-layout__background-img"></div>
+      <div className="public-layout__background-overlay"></div>
+      <Content className="public-layout__main">{children ? children : <Outlet />}</Content>
```

Comment:
[Предложение]
Для таких условий можно пользовать [оператором нулевого слияния ??](https://learn.javascript.ru/nullish-operators).
```suggestion
      <Content className="public-layout__main">{children ?? <Outlet />}</Content>
```